src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length--31 calendar--start-day--mon">
```

Comment:
Use double underscores and double hyphens for BEM naming: 'calendar__month-length--31' and 'calendar__start-day--mon'.

Reply:
You can leave lie this, yet task requirement says _implement start-day modifier_ 
Usually we use `--` to make a modifier, so it will looks like calendar--month-length--31